frontend/test/storybook/playwright.config.ts

```diff
@@ -10,6 +10,7 @@ addAliases({
 })
 
 const config: PlaywrightTestConfig = {
+  forbidOnly: !!process.env.CI,
```

Comment:
Same note here as above

Reply:
`.only` is useful to have locally if you're debugging a single, specific test, for example, in debug mode, where it runs in a "headed" browser that you can poke around the context with: https://playwright.dev/docs/debug

In that setting, if you haven't selected a single test to run, the runner is going to run _everything_ in the module, requiring you to try check which test is running each time and pass it until you get to the one you want.

That's to say: `only` is useful locally, but harmful if committed and forbidding it in CI is an easy way to catch when it is committed. Additionally, something that isn't necessarily obvious, is that Playwright and Jest are distinct test runners. Even though they look very similar in API, they have certain differences that will make the `eslint-plugin-jest` incompatible (or at least flaky) with Playwright. It would be better if our configuration didn't let you commit the test selection, though, and luckily there is a Playwright-specific ESLint plugin with the same `no-focused-test` rule that the Jest plugin provides. I've opened an issue to add it here: https://github.com/WordPress/openverse/issues/2312. That would give us a whole host of helpful checks, but also prevent `.only` from being committed at all in our Playwright tests (rather than just failing in CI).

Reply:
Woah, okay that makes sense! Thanks for the helpful context!